CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -37,1064 +37,1065 @@
       </trans-unit>
       <trans-unit id="%lld years old" xml:space="preserve">
-        <source>%lld years old</source>
+        <source>%lld years old</source><target>%lld років</target>
         <note>How old a person is right now</note>
       </trans-unit>
       <trans-unit id="%lld — " xml:space="preserve">
-        <source>%lld — </source>
+        <source>%lld — </source><target>%lld — </target>
         <note/>
       </trans-unit>
       <trans-unit id="About Callsheet" xml:space="preserve">
-        <source>About Callsheet</source>
+        <source>About Callsheet</source><target>Про Callsheet</target>
         <note/>
       </trans-unit>
       <trans-unit id="Add Pin" xml:space="preserve">
-        <source>Add Pin</source>
+        <source>Add Pin</source><target>Зберегти</target>
         <note/>
       </trans-unit>
       <trans-unit id="Add pin" xml:space="preserve">
-        <source>Add pin</source>
+        <source>Add pin</source><target>Зберегти</target>
         <note/>
       </trans-unit>
       <trans-unit id="Ages" xml:space="preserve">
-        <source>Ages</source>
+        <source>Ages</source><target>Віки</target>
         <note>Heading in Persnickety Preferences</note>
       </trans-unit>
       <trans-unit id="All" xml:space="preserve">
-        <source>All</source>
+        <source>All</source><target>Усі</target>
         <note/>
       </trans-unit>
       <trans-unit id="All Messages" xml:space="preserve">
-        <source>All Messages</source>
+        <source>All Messages</source><target>Усі повідомлення</target>
         <note>Shown in the context of a semi-hidden debugging screen.</note>
       </trans-unit>
       <trans-unit id="App Icon" xml:space="preserve">
-        <source>App Icon</source>
+        <source>App Icon</source><target>Іконка програми</target>
         <note/>
       </trans-unit>
       <trans-unit id="Are you sure?" xml:space="preserve">
-        <source>Are you sure?</source>
+        <source>Are you sure?</source><target>Ви впевнені?</target>
         <note>Shown when clearing recent searches or search history</note>
       </trans-unit>
       <trans-unit id="As rated by users of" xml:space="preserve">
-        <source>As rated by users of</source>
+        <source>As rated by users of</source><target>За оцінками користувачів</target>
         <note>Shown in a popover explaining TV/movie scores</note>
       </trans-unit>
       <trans-unit id="Average runtime" xml:space="preserve">
-        <source>Average runtime</source>
+        <source>Average runtime</source><target>Середня тривалість</target>
         <note>How long episodes run in a single season of TV</note>
       </trans-unit>
       <trans-unit id="Be a hero and support independent apps" xml:space="preserve">
-        <source>Be a hero and support independent apps</source>
+        <source>Be a hero and support independent apps</source><target>Підтримайте незалежні програми</target>
```

Comment:
Unsure about using "hero" as-is because it has a heavier meaning in current day Ukraine, so I've translated only "support independent apps" part

Reply:
100%. This is exactly in the spirit of what I was hoping for here: I'm less concerned about a verbatim translation but would actually a prefer a "here's the equivalent idea/feeling in my language".

And, about this specifically, I'm glad you made that change. It's **far** better given the awful things happening to the Ukraine right now. 💙 💛